man/runc-kill.8.md

```diff
@@ -7,9 +7,48 @@
 Where "<container-id>" is the name for the instance of the container and
 "<signal>" is the signal to be sent to the init process.
 
+The mapping between "<signal>" and the real golang signal is:
+
+    "ABRT":   syscall.SIGABRT,
```

Comment:
I'm fairly sure all of these mappings are the same as in `signal(7)`. Maybe we should just reference that?